buildSrc/src/main/kotlin/partiql.versions.kt

```diff
@@ -45,10 +45,10 @@ object Versions {
     const val kotlinxCollections = "0.3.5"
     const val picoCli = "4.7.0"
     const val kasechange = "1.3.0"
-    const val ktlint = "11.6.0"
```

Comment:
This was previously unused. The actual ktlint gradle plugin version used was defined in `buildSrc/build.gradle.kts` as 10.2.1.